mysite/profile/tests.py

```diff
@@ -1565,12 +1566,8 @@ def create_forwarder(address, valid, new_enough_for_display):
             expires_on_future_number = valid and 1 or -1
             stops_being_listed_on_future_number = (new_enough_for_display
                                                    and 1 or -1)
-            expires_on = (datetime.datetime.utcnow() +
-                          expires_on_future_number *
-                          datetime.timedelta(minutes=10))
-            stops_being_listed_on = (datetime.datetime.utcnow() +
-                                     stops_being_listed_on_future_number *
-                                     datetime.timedelta(minutes=10))
```

Comment:
Well that's easy. I can revert this change back since now that I'm looking at it more closely, I think you are correct :D